app/src/main/java/space/taran/arkmemo/data/ResourceMeta.kt

```diff
@@ -3,7 +3,7 @@ package space.taran.arkmemo.data
 import java.nio.file.attribute.FileTime
 
 data class ResourceMeta(
-    val id: Long,
+    val id: String,
```

Comment:
Actually it should be `ResourceId` from https://github.com/ARK-Builders/arklib-android/